pkg/parser/testdata/invalid-upload-caches-nonexistent-cache.yml

```diff
@@ -0,0 +1,8 @@
+container:
+  image: debian:latest
+
+task:
+  node_modules_cache:
+    folder: node_modules
+  upload_caches:
+    - mode_nodules
```

Comment:
How about a bit more complex test case like:

```yaml
container:
  image: debian:latest

task:
  binaries_cache:
    folder: binaries
  node_modules_cache:
    folder: node_modules
  upload_caches:
    - mode_nodules
   script: ./.ci/test.sh
 ```

To make sure `binaries` cache will be uploaded in the end.